pkg/services/keycloak.go

```diff
@@ -553,29 +553,12 @@ func (kc *keycloakService) registerAgentServiceAccount(clusterId string, service
 		return nil, errors.NewWithCause(errors.ErrorGeneral, tokenErr, "failed to register agent service account")
 	}
 
-	role, err := kc.getRealmRole(accessToken, roleName)
-	if err != nil {
-		return nil, err
-	}
-
-	protocolMapper := kc.kcClient.CreateProtocolMapperConfig(clusterId)
-
-	attributes := map[string]string{
-		clusterId: agentClusterId,
-	}
-
-	if kc.GetConfig().KeycloakClientExpire {
```

Comment:
let's make sure the `KeycloakClientExpire` config is removed if it is not used anywhere else

Reply:
Are we getting rid of auto expiring clients that were used for integration tests?

Reply:
> Are we getting rid of auto expiring clients that were used for integration tests?

/cc @akoserwal @ziccardi

Reply:
Thanks @akoserwal